app/templates/buyers/cancel_brief.html

```diff
@@ -7,11 +7,33 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = breadcrumbs
-  %}
-    {% include "toolkit/breadcrumb.html" %}
-  {% endwith %}
+  {{ govukBreadcrumbs({
+    "items": [
+        {
+          "href": "/",
+          "text": "Digital Marketplace"
+      },
+      {
+          "href": url_for("buyers.buyer_dashboard"),
+          "text": "Your account"
+      },
+      {
+          "href": url_for("buyers.buyer_dos_requirements"),
+          "text": "Your requirements"
+      },
+      {
+        "href": url_for(
+            ".view_brief_overview",
+            framework_slug=brief['frameworkSlug'],
+            lot_slug=brief['lotSlug'],
+            brief_id=brief['id']),
+        "text": brief['title']
+      },
+      {
+          "text": form.cancel_reason.label.text
```

Comment:
In the case when you're in the award flow, responding as to why you didn't award a brief, this is quite cluttered. It would be nice if the final breadcrumb read `Why didn't you award a contract?`, but that might be difficult to make happen given the way this view is constructed.

![image](https://user-images.githubusercontent.com/503614/72894621-9dd6e280-3d13-11ea-8f1f-4532d63f2f1a.png)

Reply:
Done too.

Reply:
This doesn't work because the view and template are used for two different pages :(

https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/db93e3b565b035c224bbf118f52eeaa445d04713/app/main/views/outcome.py#L155-L240

It might be a pain to have different breadcrumbs to the page heading for this page...

![image](https://user-images.githubusercontent.com/503614/72904538-f82d6e80-3d26-11ea-9873-b20f7c1e470c.png)

Reply:
@lfdebrux / @NoraGDS So use the heading? or the heading without the service name?

Reply:
If possible use the heading without the service name, otherwise just use the heading as before

Reply:
In this case it would be: Why do you need to cancel?

Reply:
Thanks, coming in the next commit.